playground/run.sh

```diff
@@ -1,2 +1,14 @@
 #!/bin/bash
-python -m llama_cpp.server --model ${MODEL_PATH} --host ${HOST:=0.0.0.0} --port ${PORT:=8001} --n_gpu_layers 0
+
+if [ ${CONFIG_PATH} ] || [[ ${MODEL_PATH} && ${CONFIG_PATH} ]]; then
```

Comment:
What do you mean by `[[ ${MODEL_PATH} && ${CONFIG_PATH} ]]`, if both are set you want to use `CONFIG_PATH`? IMO this case should be an error because the user has no clear idea what the behavior is going to be if both are given. How about the following instead?

```bash
if [[ -n "${CONFIG_PATH}" ]] && [[ -n "${MODEL_PATH}" ]]; then
  echo "Error: both CONFIG_PATH and MODEL_PATH cannot be set, choose only one"
  exit 1
fi

if [[ -n "${CONFIG_PATH}" ]]; then
    python -m llama_cpp.server --config_file "${CONFIG_PATH}"
elif [[ -n "${MODEL_PATH}" ]]; then
    python -m llama_cpp.server --model "${MODEL_PATH}" --host "${HOST:=0.0.0.0}" --port "${PORT:=8001}" --n_gpu_layers 0
else
    echo "Error: Either CONFIG_PATH or MODEL_PATH must be set."
    exit 1
fi
```

Reply:
The current behavior of `llama_cpp.server` is that it will "work" if both parameters are provided, but it will default to the config if both were given. So my goal was to just pass the existing behavior through. 

Also, this is the main script for running the playground on AI studio, so I wanted to be as permissive as possible to not break anything existing . 

That said, I think your suggestion is probably more user friendly. Happy to implement if others think it will work fine for AI studio.